robottelo/constants.py

```diff
@@ -1797,3 +1797,59 @@
 }
 
 RHSSO_RESET_PASSWORD = {"temporary": "false", "type": "password", "value": ""}
+
+FOREMAN_ANSIBLE_MODULES = [
+    'foreman_architecture',
```

Comment:
I know this is the way we have historically approached this, but I dislike how it mandates manual intervention to cover new modules as they appear.

Would it be possible to obtain list of all modules dynamically, from package itself or Satellite API?
This way we would cover all modules that there are automatically.

We can keep constant as "bare minimum that must exist". This will allow us to detect possibly unintentional removal of modules. We would remove it from constant only once we are certain this removal is intended.

Reply:
thanks, updated so that the list is obtained from the directory where the package installs is (more elegant solution not at hand atm)